parts/kubernetesmasteraddons-heapster-deployment.yaml

```diff
@@ -40,11 +40,9 @@ spec:
         - "--extra-cpu=0.5m"
         - "--memory=140Mi"
         - "--extra-memory=4Mi"
-        - "--threshold=5"
```

Comment:
Also can you let me know the changelog of features between 1.7 and 2.0 for the resizer?